setup.py

```diff
@@ -6,7 +6,7 @@
 
 
 def readme():
-    with open('README.md') as f:
+    with open('README.md', encoding="utf8") as f:
```

Comment:
That's an unnecessary change, imo :thinking:

Reply:
Was giving me error because sometimes python tried opening it with other encoding, don't even know why tbh

Reply:
Ach, I also misread the change - I thought you were removing it. 
Adding it is fine - I've gotten into the habit of always explicitly stating the encoding since I wrote this code.